ccpp/data/GFS_typedefs.F90

```diff
@@ -3790,6 +3798,9 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%doGP_cldoptics_PADE = doGP_cldoptics_PADE
     Model%doGP_cldoptics_LUT  = doGP_cldoptics_LUT
     Model%use_LW_jacobian     = use_LW_jacobian
+    Model%damp_LW_fluxadj     = damp_LW_fluxadj
+    Model%lfnc_k              = 1. / lfnc_k
```

Comment:
Now this is pretty confusing ... why can't `lfnc_k` be defined such that `Model%lfnc_k = lfnc_k`?  And would a default value of `1.0/-999.0=-0.001...` still be correct (I assume unused if `< 0`?)

Reply:
You're spot on. I just cleaned this up